src/components/branch-merger/components/UpdateBranchButton.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { MdUpdate, MdUpdateDisabled } from 'react-icons/md';
+import IconButton from '@material-ui/core/IconButton';
+import Badge from '@material-ui/core/Badge';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { Tooltip } from '@material-ui/core';
+
+export function UpdateBranchButton({
+  onClick = () => undefined,
+  onUpdated = () => undefined,
```

Comment:
Can we remove this prop?

Reply:
Oh, I should have checked this one too, thanks @jincypjose, I have removed it.